sdk/core/azure-core/doc/azure.core.rst

```diff
@@ -18,39 +18,39 @@ Subpackages
 Submodules
 ----------
 
-azure.core.async\_paging module
--------------------------------
+azure.core.async\_paging
+------------------------
```

Comment:
AFAIK the only time  it is necessary to escape underscores is if they happen to be trailing (because trailing underscores signify references to Sphinx)

Reply:
THat's how Sphiinx API doc generated it :p